eclipse-collections-code-generator/src/main/resources/impl/map/immutable/immutablePrimitiveObjectEmptyMap.stg

```diff
@@ -584,6 +586,12 @@ final class Immutable<name>ObjectEmptyMap\<V> implements Immutable<name>ObjectMa
         return SortedMaps.mutable.with();
     }
 
+    @Override
+    public \<NK, NV> MutableBiMap\<NK, NV> toBiMap(Function\<? super V, ? extends NK> keyFunction, Function\<? super V, ? extends NV> valueFunction)
+    {
+        return BiMaps.mutable.with();
```

Comment:
I think that is the custom followed here and `with()` in turn calls `empty()`

Reply:
we delegate with() to empty(). empty() came after with() and we kept with() around to not break users. Since empty() came we always use empty() as a convention.